docs/data/date-pickers/date-picker/DatePickerYearsOrder.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import dayjs from 'dayjs';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+
+const currentYear = dayjs();
+
+export default function DatePickerYearsOrder() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DemoContainer components={['DatePicker']}>
+        <DatePicker
+          label="Years in descending order"
+          maxDate={currentYear}
+          openTo="year"
+          views={['year', 'month']}
+          yearsOrder="desc"
+        />
+      </DemoContainer>
```

Comment:
Could we increase the min width here to avoid the clashing of a long label and the icon?
![Screenshot 2024-09-19 at 12 11 21](https://github.com/user-attachments/assets/6ade7deb-4c25-4ca5-8fc5-2569abd51278)

```suggestion
      <DatePicker
        label="Years in descending order"
        maxDate={currentYear}
        openTo="year"
        views={['year', 'month']}
        yearsOrder="desc"
        sx={{ minWidth: 250 }}
      />
```

Reply:
Good eye! I hadn't noticed this after updating the text. 🙏